server/modules/record/manager/_recordManager/nodeUpdateDependentManager.js

```diff
@@ -27,32 +25,40 @@ const logExpressionError = ({ error, expressionType, survey, nodeDef, expression
  * Module responsible for updating applicable and default values.
  */
 
-export const updateSelfAndDependentsApplicable = async (survey, record, node, tx) => {
+export const updateSelfAndDependentsApplicable = ({ survey, record, node }) => {
```

Comment:
do you need to pass the transaction?

Reply:
no! the functions in this module don't need the transaction anymore: they update the passed record in memory. They could be moved to inner functions of the RecordUpdater module.